internal/controller/operatorconfigmap_controller.go

```diff
@@ -469,21 +472,42 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI(storageClients *v1al
 				DomainLabels: slices.Collect(maps.Keys(topologyDomainLablesSet)),
 			}
 		}
+		driverSpecDefaults.GenerateOMapInfo = ptr.To(c.shouldGenerateRBDOmapInfo())
```

Comment:
@Madhu-1 do we have any side-effects if we set this flag for every driver?

Reply:
It only set on RBD driver, https://github.com/ceph/ceph-csi-operator/blob/main/internal/controller/driver_controller.go#L842

Reply:
i think no side effect (only testing can confirm it)

Reply:
Testing looks good https://github.com/red-hat-storage/ocs-client-operator/pull/411#issuecomment-2987870075